staff.html

```diff
@@ -119,6 +120,19 @@ <h2 class="is-size-2">Code Counselors</h2>
                     </div>
                 </div>
             </section> 
+
+            <!-- MODALS -->
+            <!-- NOTE:  Modals *must* be in the same order as the staff cards above to ensure the right modals become linked to the right card (i.e Brussels's modal must come before Fede's) 
+                        this does not include any staff cards that do not have modals. -->
+            <section class="modals">
+                <div class="modal">
+                    <div class="modal-background"></div>
+                    <div class="modal-content">
+                        <p>Lyall#0001</p>
+                    </div>
+                    <a class="modal-close is-large" aria-label="close"></a>
+                  </div>
+            </section>
```

Comment:
The repeated code that we're going to need to have all these modals (and also the header/footer code on each page) makes me wonder if there's a way we can use a templating engine combined with GitHub Pages and Actions.  I'll take a look into it, but this works for now.

Reply:
Yeah that would be great, at the moment it's not ideal to have to change the header six different times every time a new page is added...